nursery/encrypt-data-using-aes.yml

```diff
@@ -13,20 +13,31 @@ rule:
       - Defense Evasion::Obfuscated Files or Information::Encryption-Standard Algorithm [E1027.m05]
       - Cryptography::Encrypt Data::AES [C0027.001]
     references:
-      - https://github.com/JusticeRage/Manalyze/blob/8e77642c911d5d82b5f43b198667ab8c77a88763/bin/yara_rules/findcrypt.yara#L360
+      - https://github.com/JusticeRage/Manalyze/blob/8e77642c911d5d82b5f43b198667ab8c77a88763/bin/yara_rules/findcrypt.yara#L351
+      - https://github.com/creaktive/tsh/blob/53b822b9a07d8cc65f1f31c915cf834a2944e833/aes.c
     examples:
       - D6EFF9EFA6F93CDE95E7A4194C1BC6EE:0x180002F50
   features:
     - or:
-      - bytes: 63 7C 77 7B F2 6B 6F C5 30 01 67 2B FE D7 AB 76 = AES_SE
```

Comment:
made these lowercase and clarified the description

Reply:
Is lowercase required? Is this something that should be linted/documented?

Reply:
lower and upper case work, just wanted to make it consistent here

Reply:
see https://github.com/mandiant/capa-rules/issues/746